modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -220,6 +222,12 @@ function testSurveyAccountsAddSurvey()
 
         // Ensure that an instrument must be supplied.
         $this->safeGet($this->url . "/survey_accounts/");
+        $this->webDriver->wait(10, 1000)->until(
```

Comment:
Without this code the ajax request has not completed and the assertion fails (since the upgrade).